opcplc/OpcApplicationConfiguration.cs

```diff
@@ -33,7 +33,7 @@ public static string Hostname
         public static string ApplicationName => ProgramName;
         public static string ApplicationUri => $"urn:{ProgramName}:{HostnameLabel}{(string.IsNullOrEmpty(ServerPath) ? string.Empty : (ServerPath.StartsWith("/") ? string.Empty : ":"))}{ServerPath.Replace("/", ":")}";
         public static string ProductUri => $"https://github.com/azure-samples/iot-edge-opc-plc";
-        public static ushort ServerPort { get; set; } = 50000;
+        public static ushort ServerPort { get; set; } = 50001;
         public static string ServerPath { get; set; } = string.Empty;
```

Comment:
what was the reason for this change? there are other samples, which rely on the value 50000 as default.